app/javascript/application.js

```diff
@@ -1,3 +1,21 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
+import Rails from "@rails/ujs"
+Rails.start()
+
+//= require rails-ujs
+//= require turbolinks
+//= require_tree .
+
+
+
+
+//= require jquery3
+//= require rails-ujs
+
+//= require jquery3
+//= require popper
+//= require bootstrap
+//= require rails-ujs
+//= require_tree .
```

Comment:
if thise are commented out  then remove them. if not why we have duplicate lines here?